packages/graphiql/README.md

```diff
@@ -6,11 +6,109 @@
 
 _/ˈɡrafək(ə)l/_ A graphical interactive in-browser GraphQL IDE. [Try the live demo](http://graphql.org/swapi-graphql).
 
-[![](resources/graphiql.png)](http://graphql.org/swapi-graphql)
+[![](https://raw.githubusercontent.com/graphql/graphiql/HEAD/resources/graphiql.png)](http://graphql.org/swapi-graphql)
 
-### Getting started
+## Features
 
-Using a node.js server? Just use [`express-graphql`](https://github.com/graphql/express-graphql)! It can automatically present GraphiQL. Using another GraphQL service? GraphiQL is pretty easy to set up. With `npm`:
+- Syntax highlighting.
+- Intelligent type ahead of fields, arguments, types, and more.
+- Real-time error highlighting and reporting for queries and variables.
+- Automatic query and variables completion.
+- Automatically adds required fields to queries.
+- Documentation explorer, search, with markdown support.
+- Query History using local storage
+- Run and inspect query results using _any_ promise that resolves JSON results. HTTPS or WSS not required.
+- Supports full [GraphQL Language Specification](https://github.com/graphql/graphql-wg): 
+  - Queries, Mutations, Subscriptions, Fragments, Unions, directives, multiple operations per query, etc
+
+
+## Demos
+
+We have a few demos of `master` branch via the default netlify build (the same URL paths apply to deploy previews on PRs):
+
+1. [`graphiql.min.js` demo](https://graphiql-test.netlify.com/) - the min.js bundle for the current ref
```

Comment:
these demo url paths will work after merge. use the paths with the deploy preview to see them in actuon